### PR TITLE
featured SubQuery

### PR DESCRIPTION
**Pull request by bgodlin**
Hello Polygon team,

We are submitting this contribution with the intention of incorporating SubQuery into your project documentation. As you can observe, we have already prepared the initial project. Kindly inform us if any adjustments are necessary on our part.

### CODE REVIEW

**Review by rahuldamodar94 — APPROVED** [review]

File: docs/tools/data/subquery.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- [Polygon ZKEVM Quick Start Guide](https://academy.subquery.network/quickstart/quickstart_chains/polygon-zkevm.html)
+
+- [Polygon Plasma Bridge Multi-Chain Quick Start Guide](https://academy.subquery.network/quickstart/quickstart_multichain/polygon-plasma-bridge.html)
+
+- [Polygon's Starter Projects Directory](https://github.com/subquery/ethereum-subql-starter/tree/main/Polygon)
+
+For technical questions and support reach out to us start@subquery.network
+
+## Running and Hosting your Polygon SubQuery APIs
```
Comment (kmurphypolygon) [inline docs/tools/data/subquery.md]:
```suggestion
## Running and hosting your Polygon SubQuery APIs
```

File: docs/tools/data/subquery.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Intro Quick Start Guide: [1. Create a New Project](https://academy.subquery.network/quickstart/quickstart.html)
+
+- [Polygon Quick Start Guide](https://academy.subquery.network/quickstart/quickstart_chains/polygon.html)
+
+- [Polygon ZKEVM Quick Start Guide](https://academy.subquery.network/quickstart/quickstart_chains/polygon-zkevm.html)
+
+- [Polygon Plasma Bridge Multi-Chain Quick Start Guide](https://academy.subquery.network/quickstart/quickstart_multichain/polygon-plasma-bridge.html)
+
+- [Polygon's Starter Projects Directory](https://github.com/subquery/ethereum-subql-starter/tree/main/Polygon)
```
Comment (kmurphypolygon) [inline docs/tools/data/subquery.md]:
```suggestion
- SubQuery docs: [SubQuery Academy (Documentation)](https://academy.subquery.network/)

- Intro quick start guide: [1. Create a New Project](https://academy.subquery.network/quickstart/quickstart.html)

- [Polygon Quick Start Guide](https://academy.subquery.network/quickstart/quickstart_chains/polygon.html)

- [Polygon ZKEVM quick start guide](https://academy.subquery.network/quickstart/quickstart_chains/polygon-zkevm.html)

- [Polygon Plasma bridge multi-chain quick start guide](https://academy.subquery.network/quickstart/quickstart_multichain/polygon-plasma-bridge.html)

- [Polygon's starter projects directory](https://github.com/subquery/ethereum-subql-starter/tree/main/Polygon)
```

**Comment by kmurphypolygon** [comment]
Hi @bgodlin just a few stylistic suggestions. Please can you commit and we will go ahead and publish. Thank you.

**Review by kmurphypolygon — APPROVED** [review]
LGTM

**Review by kmurphypolygon — APPROVED** [review]
LGTM. Thank you for your submission :)

**Pull request merged**